main.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+cmd_build() {
+  short_hash=$(git rev-parse --short HEAD)
+
+  echo "Building container"
+  docker build \
+      -t 864879987165.dkr.ecr.us-east-1.amazonaws.com/calm/localstripe:latest \
```

Comment:
can you only put latest for master branch instead of all builds.

Reply:
I was planning on keeping `master` clean so it can merge from/to upstream if needed, and keep calm-specific stuff on `calm` branch, so I added logic to only tag `latest` on `calm` branch - let me know if this will be an issue